src/surge-python/setup.py

```diff
@@ -26,6 +26,6 @@ def just_surgepy(cmake_manifest):
     install_requires=["numpy"],
     packages=["surgepy"],
     cmake_source_dir="../..",
-    cmake_args=["-DSURGE_BUILD_PYTHON_BINDINGS=TRUE"],
+    cmake_args=["-DSURGE_BUILD_PYTHON_BINDINGS=TRUE", "-DSURGE_SKIP_STANDALONE=TRUE"],
```

Comment:
I'm happy to merge this but I bet we want to include some of the other 'skip' options we put in place for rack and others, like SKIP_VST3 

https://github.com/surge-synthesizer/surge-rack/blob/9f9922ddbc234fc89d4b6c1b8be4364dec72af40/CMakeLists.txt#L24

here's the list we use in rack. For python I bet we can do

```
set(SURGE_SKIP_JUCE_FOR_RACK TRUE CACHE BOOL "skip surge Juce build")
set(SURGE_SKIP_ODDSOUND_MTS TRUE CACHE BOOL "skip surge oddsound-mts build")
set(SURGE_SKIP_VST3 TRUE CACHE BOOL "skip surge VST3 build")
set(SURGE_SKIP_ALSA TRUE CACHE BOOL "skip surge Alsa build")
set(SURGE_SKIP_STANDALONE TRUE CACHE BOOL "skip surge standalone build")
```

and still hav python work just fine

Reply:
oh let me know if you want me to merge and then test this later or if you prefer to change and test and then have it merged

Reply:
Thanks very much! Skipping more would be great. I suspect it would speed up building the surgepy Python package. It might also mean we can drop some of the 'yum install' packages for building the wheels; they were just for juce stuff if I remember.

I'll try adding those skip flags and retest - can do it as part of this PR.

Reply:
great thanks!

the skip juce one makes a new library target you need to include called l`juce_dsp_rack_sub` which depending on the linkage pattern of surge python you may need to add explicitly to the link phase when juce is off.

Reply:
No problem at all. I pushed a version with those skip flags added and all the yum install packages removed. The surgepy Python package still builds and passes the surgepy tests, and the cibuildwheel wheel building still works. It's also easily twice as fast to build now!

Reply:
Awesome. Will merge now